### PR TITLE
Update WAFS tag and increase number of tasks for wafs_grib2_0p25

### PR DESCRIPTION
**Pull request by YaliMao-NOAA**
1. Update WAFS tag with adding forecast hour window to dbn_alert for awf and WAFS unblended hazard data

2. Increase number of tasks for wafs_grib2_0p25

Refs #2013

# Description

# Type of change

- Bug fix (fixes something broken)
- New feature (adds functionality)
- Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? YES/NO
- Does this change require a documentation update? YES/NO

# How has this been tested?

# Checklist
- [ ] Any dependent changes have been merged and published
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Review by KateFriedman-NOAA — APPROVED** [review]
Changes are what was discussed with NCO SPA. Thanks @YaliMao-NOAA ! Approving and then merging before cutting updated tag.

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:30:00
-#PBS -l select=1:mpiprocs=11:ompthreads=1:ncpus=11:mem=80GB
+#PBS -l select=1:mpiprocs=39:ompthreads=1:ncpus=39:mem=200GB
```
Comment (aerorahul) [inline ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf:7]: I am sure NCO will ask why we are increasing this and why it is not reflected in the Release notes.
@YaliMao-NOAA Can you please justify these increases?

Comment (YaliMao-NOAA) [reply ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf:7]: @aerorahul Xiaoxue, who is in charge of this implementation, pointed out the change that I missed in the previous PR. Should we still include in the release notes?

Comment (aerorahul) [reply ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf:7]: It is a good idea to make a note in the release notes as we document all changes from the previous release.

Comment (YaliMao-NOAA) [reply ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf:7]: @KateFriedman-NOAA Sorry I didn't think about the whole situation throughly. I made another PR #2050 for the release note update.

**Pull request merged**